boards/iot-lab_M3/include/ng_at86rf2xx_params.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2015 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup   board_iot-lab_M3
+ * @brief     at86rf231 board specific configuration
```

Comment:
`brief` should go after `file`. And some newlines to fit the formatting of all the other RIOT headers would be nice.